### PR TITLE
style: minor formatting and capitalisation changes

### PR DESCRIPTION
**Pull request by PaulL1**
Various changes - adding semicolons, compilation fixes, capitalisation/standards changes.

### CODE REVIEW

File: lib/ngbp/cli.js [hunk 1]
```diff
@@ -130,10 +130,11 @@ function processOptions () {
...
-  var contents
+  var contents;
+  var path;
 
   ASYNC.each( files, function ( file, cb ) {
-    FS.readFile( file, function ( err, contents ) {
+    FS.readFile( file, function ( err, data ) {
       if ( ! err ) {
         exists = true;
         contents = contents;
```
Comment (joshdmiller) [inline lib/ngbp/cli.js]: If you rename `contents` to `data`, it has to be fixed here too, no?

**Comment by joshdmiller** [comment]
Holy crap I'm sloppy...

File: lib/ngbp/cli.js (hunk 1 above)
Comment (PaulL1) [inline lib/ngbp/cli.js]: Yes.  That was sloppy on my part.  :-)  Will submit an update.

**Comment by joshdmiller** [comment]
Landed as acd6378. Thank you, sir!

**Pull request closed without merging**